packages/gatsby-theme-styleguide/src/templates/pattern-page.js

```diff
@@ -108,10 +149,34 @@ export default ({ data, location }) => {
   const hasTabs = samePageTabs.length > 1
   const hasMainCategory = mainCategory.length > 3
 
+  function handleMenu(e) {
```

Comment:
The params `e` event is passed but never used.

Reply:
fixed